examples/batch_processing.ipynb

```diff
@@ -0,0 +1,1148 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "51c7a1f9",
+   "metadata": {},
+   "source": [
+    "# Batch processing with the Batch API\n",
+    "\n",
+    "The new Batch API allows to **create async batch jobs for a lower price and with higher rate limits**.\n",
+    "\n",
+    "Jobs will be completed within 24h, but can be completed faster depending on global usage. \n",
+    "\n",
+    "This notebook covers how to use the Batch API with a practical example.\n",
+    "\n",
+    "As an example, we will caption images using the Amazon furniture dataset, using the `gpt-4-vision-preview` model. \n",
+    "\n",
+    "Please note that multiple models are available through the Batch API, and that you can use several parameters in your batch API calls, as you would with the Chat Completions endpoint."
```

Comment:
should "several" be "the same"

consistency on capitalizing "batch"

Reply:
Will make the changes!